### DISCUSSION TITLE
README section for Ethereum.world instance

### DISCUSSION THREAD

**Discussion by josenriagu** [AKASHAorg/akasha-core#1161] [General]
I think we should have a distinct section in the root README for Ethereum.world app

**Comment by felixboehm** [comment]
It would be good to cover the development getting started. It makes sense to direct that to the ethereum.world example, as it is most prominent.

Some input
- npm version needs to match. newest npm version is not building, with unresolved conflicts.
- configurations need explanation. I could not find hints on how to create a development config with a working invite service for signups.

**Reply by kenshyx** [reply]
Hey @felixboehm, thank you for your feedback! 

Regarding your issues with npm, we've applied a patch this week that enables successful builds with `npm v8.1.2`  and `node v16.13.0`. We're running some test builds on circleCI with this configuration. Can you give it a try again? If the problem persists it would be great to have this captured in a issue on github or leave us here your configuration. 
We will also update the github documentation as soon as possible!

Thank you for your interest in building ethereum world together! 🍻
